### PR TITLE
Adds a remote environment for DuckDB databases running in a Buena Vista server

### PR DESCRIPTION
**Pull request by jwills**
(using the postgres protocol)

### CODE REVIEW

File: dbt/adapters/duckdb/environments.py [hunk 1]
```diff
@@ -41,50 +45,101 @@ def handle(self):
...
         # Extensions/settings need to be configured per cursor
-        cursor = self.conn.cursor()
-        for ext in self.creds.extensions or []:
+        for ext in creds.extensions or []:
             cursor.execute(f"LOAD '{ext}'")
-        for key, value in self.creds.load_settings().items():
+        for key, value in creds.load_settings().items():
             # Okay to set these as strings because DuckDB will cast them
             # to the correct type
             cursor.execute(f"SET {key} = '{value}'")
```
Comment (Mause) [inline dbt/adapters/duckdb/environments.py:88]: Might be worth escaping quotes here?

Comment (jwills) [reply dbt/adapters/duckdb/environments.py:88]: I don't understand?

Comment (jwills) [reply dbt/adapters/duckdb/environments.py:88]: (Like, which quotes should I be escaping?)

Comment (Mause) [reply dbt/adapters/duckdb/environments.py:88]: In the case that someone passes a string containing single quotes I mean - they should be doubled up to escape them. It's not directly relevant to this pr, just figured it was worth mentioning

**Pull request merged**